<?xml version="1.0"?>

<launch>
  <arg name="model_path" doc="The absolute path to the weights of the network."/>
  <arg name="namespace" default=""/>
  <group ns="/$(arg namespace)">
    <node pkg="dreamer2ros" name="deploy_img2img" type="img2img_deploy.py" output="screen">
         <param name="model_path" type="string" value="$(arg model_path)"/>
         <remap from="reward_generator/DreamersView" to="reward_follow_shore/DreamersView"/>
    </node>
  </group>
</launch>
